src/ansys/pyensight/core/utils/omniverse_dsg_server.py

```diff
@@ -299,16 +299,10 @@ def create_dsg_mesh_block(
             mesh.CreateFaceVertexCountsAttr([3] * (conn.size // 3))
             mesh.CreateFaceVertexIndicesAttr(conn)
             if (tcoords is not None) and variable:
-                # USD 22.08 changed the primvar API

```

Comment:
The version of the Usd module is controlled by the wheel dependencies, so we can remove backward compatibility.